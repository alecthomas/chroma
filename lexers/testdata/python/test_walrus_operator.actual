if (a := 2) > 4:
