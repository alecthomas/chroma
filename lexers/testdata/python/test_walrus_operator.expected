[
  {"type":"Keyword","value":"if"},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"("},
  {"type":"Name","value":"a"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":":="},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"2"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"\u003e"},
  {"type":"Text","value":" "},
  {"type":"LiteralNumberInteger","value":"4"},
  {"type":"Punctuation","value":":"},
  {"type":"Text","value":"\n"}
]
